duneggd/Config/TMS.cfg

```diff
@@ -1,10 +1,81 @@
 [TMS]
 class 	= 	duneggd.SubDetector.TMS.tmsBuilder
 mat   	= 	"SteelTMS"
+#for 7m width
 thinbox1Dimension = [Q('1.850m'),Q('4.7m'),Q('0.015m')]
+#for 6m width 
+#thinbox1Dimension = [Q('1.4575m'),Q('4.7m'),Q('0.015m')]
+
 #thinbox2Dimension = [Q('3.498m'),Q('5.022m'),Q('0.015m')]  # #remnant from old geometry version with two different steel plate widths (outer and inner)
 gapPosition =  [Q('0.020m'),Q('0m'),Q('0m')]
-BFieldUpLow = '(0 T, -1 T, 0 T)'#(0 T,3 T,0 T)
-BFieldUpHigh = '(0.0 T, -1 T, 0 T)'#(0.0 T, 5.5 T, 0 T)
-BFieldDownLow = '(0 T, 1 T, 0 T)'#(0 T, -3 T, 0 T)
-BFieldDownHigh = '(0.0 T,  1 T, 0 T)'#(0.0 T, -5.5 T, 0 T)
+BFieldUpLow = '(0 T, -1.4 T, 0 T)'#(0 T,3 T,0 T)
+BFieldUpHigh = '(0.0 T, -1.4 T, 0 T)'#(0.0 T, 5.5 T, 0 T)
+BFieldDownLow = '(0 T, 1.4 T, 0 T)'#(0 T, -3 T, 0 T)
+BFieldDownHigh = '(0.0 T,  1.4 T, 0 T)'#(0.0 T, -5.5 T, 0 T)
+
+
+Module_layers_thin=34
+Module_layers_thick=22
+Module_layers_double=25
+
+# c,d geometries
+thin_horizontal = [1, 4, 7, 14, 21, 28]
+thick_horizontal  = [0, 7, 14,21] 
+double_horizontal = [6, 13,20,24]
+thin_U    = [2, 5, 8, 10, 12, 15, 17, 19, 22, 24, 26, 29, 31, 33]
+thick_U   = [1, 3, 5, 8, 10, 12, 15, 17, 19]
+double_U  = [0, 2, 4, 7, 9, 11, 14, 16, 18,21,23]
+thin_V    = [3, 6, 9, 11, 13, 16, 18, 20, 23, 25, 27, 30, 32, 34]
+thick_V   = [2, 4, 6, 9, 11, 13, 16, 18, 20]
+double_V  = [1, 3, 5, 8, 10, 12, 15, 17,19,  22]
+
+
+
+#Script to print module layer configuration summary with computed indices.
```

Comment:
Please don't add/keep any big blocks of commented out code that are not actually used. If you want to print out the orientation summary in the log, you can just do that. So either use this code or remove it.